driver-mapping/pom.xml

```diff
@@ -108,11 +109,37 @@
             <scope>test</scope>
         </dependency>
 
+        <dependency>
+            <groupId>org.codehaus.groovy</groupId>
+            <artifactId>groovy-all</artifactId>
+            <version>${groovy.version}</version>
+            <scope>test</scope>
+        </dependency>
```

Comment:
No, it was more to avoid downloading the internet when you first run tests. We just want to ignore a `metaClass` property, we could test it without Groovy. But I see the value in a full-fledged Groovy integration test as well. We can keep that as-is for now, and we'll reconsider the tests structure for 1296.